lnc/lnc.go

```diff
@@ -281,8 +281,12 @@ func (n *NodeConn) newConn(session *Session, opts ...grpc.DialOption) (*conn,
 	}
 	dialOpts = append(dialOpts, opts...)
 
+	ctxt, cancelT := context.WithTimeout(
+		context.Background(), DefaultConnectionTimetout,
+	)
+	defer cancelT()
```

Comment:
Are you sure we can cancel this context here? Is the context only used for the initial dialing and not for the whole duration of the connection?

Reply:
Good question, I had to check it before opening the PR because I wasn't sure neither.


Form the [grpc docs](https://pkg.go.dev/google.golang.org/grpc#DialContext):

> In the blocking case, ctx can be used to cancel or expire the pending connection. Once this function returns, the cancellation and expiration of ctx will be noop. Users should call ClientConn.Close to terminate all the pending operations after this function returns.

Deployed and tested on Friday  👍